driver/ppm_events_public.h

```diff
@@ -1313,7 +1313,9 @@ enum ppm_syscall_code {
 	PPM_SC_RENAMEAT2 = 320,
 	PPM_SC_USERFAULTFD = 321,
 	PPM_SC_OPENAT2 = 322,
-	PPM_SC_MAX = 323,
+	PPM_SC_UMOUNT2 = 323,
+	PPM_SC_EXECVE = 324,
+	PPM_SC_MAX = 325,
```

Comment:
So... how did we miss this before?

Reply:
That's quite a nice question indeed :)  
Given that routing table is only used by `PPME_GENERIC_` events to send the first argument (internal syscall ID), and that both execve and umount2 are instead parsed as specific events, we never had any issue with them missing.  
But now, routing table is also used as a translator between internal PPM_SC code and real syscall nr, for the syscalls_of_interest feature.

I double checked that no PPM_SC is missing right now.